steward/src/allocation.rs

```diff
@@ -231,12 +231,12 @@ pub async fn direct_rebalance(
 ) -> Result<(), Error> {
     let mut tick_info: Vec<CellarTickInfo> = Vec::new();
     let config = APP.config();
-    let keystore = path::Path::new(&config.keystore);
+    let keystore = path::Path::new(&config.keys.keystore);
```

Comment:
So, we used to have two keystore references: one in `config.keystore` and one in `config.keys.keystore`. It was a conscious choice in a recently merged PR to just use a global keystore config (it's necessary for every function of steward). I think the `keys` stanza in this case would just be a list of specific keys that could be in use.

Reply:
Ah, I was unaware of the discussion around it.